### ISSUE TITLE
Ping: add support for NMEA mode output

### ISSUE THREAD

**Issue by patrickelectric** [bluerobotics/ping-firmware#6]

**Comment by iain-parnum** [comment]
I would like to add my request for NMEA output from Ping. Thanks, Iain

**Comment by eabase** [comment]
> I would like to add my request for NMEA output from Ping. Thanks, Iain

I don't think you understand that sonar and GPS location are totally different things. 
Issue can be closed.
